src/assets/simp/build/simp.spec

```diff
@@ -184,12 +182,14 @@ chmod u=rwX,g=rX,o=rX -R %{buildroot}%{_sysconfdir}/simp
 # Post installation stuff
 export PATH=/opt/puppetlabs/bin:/opt/puppetlabs/puppet/bin:$PATH
 
-if [ -f /etc/puppetlabs/puppet/autosign.conf ]; then
-  chmod 644 %{prefix}/autosign.conf;
+puppet_confdir=`puppet config print confdir`
+puppet_environmentpath=`puppet config print environmentpath`
+
+if [ -f "${puppet_confdir}/autosign.conf" ]; then
+  chmod 644 "${puppet_confdir}/autosign.conf"
 fi
 
-simp_env="`puppet config print environmentpath`/simp"
-if [[ -d $simp_env && -f '%{_usr}/local/sbin/hiera_upgrade' ]]; then
+if [[ -d "${puppet_environmentpath}/simp" && -f '%{_usr}/local/sbin/hiera_upgrade' ]]; then
```

Comment:
We haven't delivered that script in a while.  Can this logic be removed?

Reply:
@lnemsick-simp If you're 100% sure, then go ahead and remove it. I wasn't so I left it.